commit 2bffdc28ee6029df705996f6b5566ccd049673c6
Author: Bart Van Assche <bart.vanassche@wdc.com>
Date:   Mon Jul 2 10:06:51 2018 -0700

    include/rdma/opa_addr.h: Fix an endianness issue
    
    [ Upstream commit 4eefd62c17a9a5e7576207e84f3d2b4f73aba750 ]
    
    IB_MULTICAST_LID_BASE is defined as follows:
    
      #define IB_MULTICAST_LID_BASE   cpu_to_be16(0xC000)
    
    Hence use be16_to_cpu() to convert it to CPU endianness. Compile-tested
    only.
    
    Fixes: af808ece5ce9 ("IB/SA: Check dlid before SA agent queries for ClassPortInfo")
    Signed-off-by: Bart Van Assche <bart.vanassche@wdc.com>
    Cc: Venkata Sandeep Dhanalakota <venkata.s.dhanalakota@intel.com>
    Cc: Mike Marciniszyn <mike.marciniszyn@intel.com>
    Cc: Dennis Dalessandro <dennis.dalessandro@intel.com>
    Signed-off-by: Jason Gunthorpe <jgg@mellanox.com>
    Signed-off-by: Sasha Levin <alexander.levin@microsoft.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

diff --git a/include/rdma/opa_addr.h b/include/rdma/opa_addr.h
index 2bbb7a67e643..66d4393d339c 100644
--- a/include/rdma/opa_addr.h
+++ b/include/rdma/opa_addr.h
@@ -120,7 +120,7 @@ static inline bool rdma_is_valid_unicast_lid(struct rdma_ah_attr *attr)
 	if (attr->type == RDMA_AH_ATTR_TYPE_IB) {
 		if (!rdma_ah_get_dlid(attr) ||
 		    rdma_ah_get_dlid(attr) >=
-		    be32_to_cpu(IB_MULTICAST_LID_BASE))
+		    be16_to_cpu(IB_MULTICAST_LID_BASE))
 			return false;
 	} else if (attr->type == RDMA_AH_ATTR_TYPE_OPA) {
 		if (!rdma_ah_get_dlid(attr) ||
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1128_2bffdc28ee6029df705996f6b5566ccd049673c6.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1128_2bffdc28ee6029df705996f6b5566ccd049673c6.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
